### ISSUE TITLE
When is uniquebodyparts appropriate?

### ISSUE THREAD

**Issue by arjunt19** [DeepLabCut/DeepLabCut#1405]
Hi, I'm using DLC for an experiment with one adult animal and multiple children. I assumed that to differentiate between the mother and the children, I could use identity: True in the config.yaml file, and labeled all the images (same set of labels for both the children and the mother). However, I just read about using uniquebodyparts - is that more appropriate? If so, do I need to relabel all of my images? Thanks for any help! Also side question - in the maDLC paper, it states that the 4 training datasets were released. Can you please inform me as to where they are? That would be a great help.

**Comment by MMathisLab** [comment]
Hi @arjunt19 it would be great to post questions on the forum, vs GitHub in the future please. 

and no, identity true is for when you have say 3 black mice but their tails are marked so you can visually tell them apart. then ID=true is good. In your case, the the mother is the unique, and the children are the multianimal bodyparts. This is like the parenting mouse example in the paper.

Datasets for maDLC **will** be released when the paper is published, so stay tuned!


More tips from this page: https://github.com/DeepLabCut/DeepLabCut/blob/master/docs/convert_maDLC.md 

Example: Imagine you had a project with the moon and a rocket with two parts labeled:
`bodyparts: [moon, rocket_tip,rocket_bottom]`

Now you want to use this former project (labeled-data) and work on a new dataset (videos) with one moon but multiple (3) rockets. Then convert it as follows:
```
individuals: [rocket1, rocket2, rocket3]
uniquebodyparts: [moon]
multianimalbodyparts: [rocket_tip,rocket_bottom]
skeleton: [[[rocket_tip,rocket_bottom]]]
```
In the unusual case, that your data also has multiple moons (e.g. is now carried out around Jupiter), but one rocket:
```
individuals: [Io, Europa, Ganymede, Callisto]
uniquebodyparts: [rocket_tip,rocket_bottom]
multianimalbodyparts: [moon]
```
Note you can use the single object tracker for this situation. What if you have multiple moons and rockets?
